src/components/mobile-menu-container/components/mobile-user-disclosure/mobile-user-disclosure.module.css

```diff
@@ -0,0 +1,99 @@
+.root {
+	/* composition */
+	composes: hds-surface-base from global;
+
+	/* properties */
+	background-color: var(--token-color-surface-faint);
+	border-radius: 6px;
+	margin-bottom: 24px;
```

Comment:
I think theres some double margin happening. Looks like its stacking with another wrapper component to be `48px. In Figma I see this space is set at `30px`.
<img width="600" alt="Screen Shot 2022-07-29 at 10 52 25 AM" src="https://user-images.githubusercontent.com/36613477/181817157-5509e464-54ea-4d46-a4b9-2713a04a0e99.png">
`

Reply:
Nice catch!! 👀 This happened when I moved the component into a wrapper, and I didn't double-check the spacing. Thank you 🤩

Reply:
Resolved in https://github.com/hashicorp/dev-portal/pull/747/commits/5c5c42cf912a7b4dd2f1d4284cae306c19fc70a2